main.js

```diff
@@ -0,0 +1,358 @@
+argument. The function should return the median number in an array. 
+Hmmmm, haven't you found a median of the array before? Maybe not..
+
+Hint: Use some kind of sorting method to put the values in order first,
+ then somehow access the median element, somehow using the length of 
+ the array....maybe? We're going to go over arrays in depth on Monday 
+ but see if you can find some Array methods that can accomplish what 
+ you need. */
+var nums =[1,14,5,27,8,9,10,39];
+
+var arr =  nums.sort(function(a, b){return a-b});
+
+function medianOfArray(nums){
+    var count = 0;
+    for(var i = 0;i<nums.length;i++){
+        count=count + nums[i];
+    }
+    mediem = count / nums.length;
+    return mediem;
+
+}
+var result = medianOfArray(arr);
+
+console.log(result)
+
+//---------------------------------
+
+/*
+PART 4: Sum Array
+Create a function called sumArray that takes an array arr as an 
+argument and returns the sum of all the numbers in the array. If the 
+array is empty then return 0 */
+var list =[1, 4, 7, 8, 9];
+function sumArray(arr){
+var sum= 0;
+    for(var i =0; i < arr.length; i++){
+        sum= sum + arr[i];
+
+    }
+    return sum;
+}
+
+var result = sumArray(list);
+console.log(result);
+
+//----------------------------------
+/* 
+PART 5: Vowel Count
+Create a function called vowelCount that accepts a str as an argument 
+and returns the number of vowels in that string. */
+
+function vowelCount(str1)
+{
+  var vowelList = 'aeiouAEIOU';
+  var vlcount = 0;
+  
+  for(var x = 0; x < str1.length ; x++)
+  {
+    if (vowelList.indexOf(str1[x]) !== -1)
+    {
+      vlcount += 1;
+    }
+  
+  }
+  return vlcount;
+}
+console.log(vowelCount("I am working on my homework"));
+
+//----------------------------------------
+
+/*
+PART 6: Initials
+Write a function called initials that accepts one argument 
+(person's name) and returns their initials.
+
+It should work both with and without middle name being provided.
+Hint! Remember, string methods? Use them!
+
+initials('Neil Patrick Harris');
+// NPH 
+*/
+
+function initials(fullname){
+    var capptel = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+  var letter = "";
+  
+  for(var x = 0; x < fullname.length ; x++)
+  {
+    if (capptel.indexOf(fullname[x]) !== -1)
+    {
+      letter += fullname[x];
+    }
+  
+  }
+  return letter;
+
+}
+console.log(initials("Nawaf Ibrahim Alhahni"));
+
+//---------------------------------------------------
+/*
+PART 7: Days of the week
+Write an conditional JavaScript statement that prompts the user to 
+specify what day is today. 📅
+
+How do you create a prompt popup in the browser? Google might know 🙃
+
+If it's Monday alert "Energize!"
+If it's Tuesday alert "Just getting started!"
+If it's Wednesday alert "Hump Hump!"
+If it's Thursday alert "Almost there!"
+If it's Friday, Saturday or Sunday alert "Weeeeeee!"
+If it's not anything above alert "Huh, sorry didn't get that?"
+*/
+
+var daysOfWeek = "Sunday";
```

Comment:
what if the `daysOfWeek = "Monday"` how to access the switch statement and print Energize! we need to use prompt to make the user enter the day